libpod/container.go

```diff
@@ -1141,8 +1141,13 @@ func (c *Container) IDMappings() (storage.IDMappingOptions, error) {
 
 // RootUID returns the root user mapping from container
 func (c *Container) RootUID() int {
+	return c.findUID(0)
+}
+
+// findUID returns the UID mapping of the specified uid or 0 if it couldn't be found.
+func (c *Container) findUID(uid int) int {
 	for _, uidmap := range c.config.IDMappings.UIDMap {
-		if uidmap.ContainerID == 0 {
+		if uidmap.ContainerID == uid {
```

Comment:
is the function supposed to be used only with `findUID(0)` as it seems in the current version?  Otherwise it needs to check if the uid is in the `[idmap.ContainerID; idmap.ContainerID+idmap.Size)` range and the offset must be added to the return value.